models/iris/iris.sdf.jinja

```diff
@@ -487,10 +490,11 @@
       <robotNamespace/>
       <imuSubTopic>/imu</imuSubTopic>
       <magSubTopic>/mag</magSubTopic>
+      <gpsSubTopic>/gps</gpsSubTopic>
```

Comment:
I think `gpsSubTopic` has been deprecated.

Reply:
This is in here to provide support for older versions and doesn't hurt to have.

Reply:
This just adds noise. Please remove it.